### ISSUE TITLE
Other Languages

### ISSUE THREAD

**Issue by Bl00d-B0b** [DevMiser/Lumina#1]
Hello,

Is it possible to have in other languages?

Ed

**Comment by DevMiser** [comment]
Yes, it is possible with a little effort.  The 'Lumina' wakeword is trained in English, but you can train other wakewords in other languages using the Picovoice console. See 
https://picovoice.ai/platform/porcupine/
For the remainder of the voice control which uses speech-to-text, you need to use the corresponding model file for the desired language.  The model files for the supported languages are on the Picovoice Github repo.  See (https://github.com/Picovoice/leopard/tree/master/lib/common)
OpenAI's API is optimized for use in English, but it is robust enough to generate decent results in other languages.
